package.json

```diff
@@ -58,25 +65,24 @@
   },
   "watches": {
     "doc": [
```

Comment:
It's a part of https://www.npmjs.com/package/rerun-script . Basically, run script `doc` when anything changes in the following globs.